src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetectionTransaction+DidEndPrematurelyReason.swift

```diff
@@ -45,11 +45,34 @@ extension ExposureDetection.DidEndPrematurelyReason: LocalizedError {
 			guard let enError = error as? ENError else {
 				return AppStrings.ExposureDetectionError.errorAlertMessage + " Code: NoSummary"
 			}
-			return AppStrings.ExposureDetectionError.errorAlertMessage + " EN Code: \(enError.code.rawValue)"
+			switch enError.code {
+			case .unsupported:
+				return AppStrings.ExposureDetectionError.enError5Description
+			case .internal:
+				return AppStrings.ExposureDetectionError.enError11Description
+			case .rateLimited:
+				return AppStrings.ExposureDetectionError.enError13Description
+			default:
+				return AppStrings.ExposureDetectionError.errorAlertMessage + " EN Code: \(enError.code.rawValue)"
+			}
 		case .noDaysAndHours:
 			return AppStrings.ExposureDetectionError.errorAlertMessage + " Code: NoDaysAndHours"
 		case .noExposureConfiguration:
 			return AppStrings.ExposureDetectionError.errorAlertMessage + " Code: NoExposureConfiguration"
 		}
 	}
 }
+
+extension ENError {
+	var faqURL: URL? {
+		switch code {
+		case .unsupported:
+			return URL(string: AppStrings.Links.appFaqENError5)
+		case .internal:
+			return URL(string: AppStrings.Links.appFaqENError11)
+		case .rateLimited:
+			return URL(string: AppStrings.Links.appFaqENError13)
+		default: return nil
+		}
+	}
+}
```

Comment:
This extension seems like it could go into a separate file, but I wasn't sure if it's worth it as it's only used here. 

It is not private as we'd like to unit test it.

Reply:
Moved this to a separate extension, and added the ExposureSubmission stuff into it as well. Check `Error+FAQUrl.swift` for this change.